addons/hw_posbox_homepage/views/layout.html

```diff
@@ -13,6 +13,7 @@
                 text-align: justify;
                 color: #6B6B6B;
                 background-color: #f1f1f1;
+                background-image: url("/hw_posbox_homepage/static/img/background-light.svg");
```

Comment:
without any extra style, this will repeat the image horizontally and vertically which is pretty ugly on > 1920x1080 resolution.
As the picture is stolen from odoo homepage, maybe we should copy the extra style as well ?

```suggestion
                background-image: url("/hw_posbox_homepage/static/img/background-light.svg");
                background-size: cover;
                background-attachment: fixed;
```
With current code:
![image](https://github.com/odoo/odoo/assets/60775325/c6b01e0c-c242-43e8-8956-f9319addda8b)

After:
![image](https://github.com/odoo/odoo/assets/60775325/60d96bee-d1ab-4bf8-a02f-273c36b9d7ce)